bucket/interfaces.go

```diff
@@ -0,0 +1,23 @@
+package bucket
+
+import (
+	"net/http"
+	"fmt"
+	"github.com/mateusz/tempomat/lib/config"
+)
+
+
+type Entry interface {
+	fmt.Stringer
+	Credits() float64
+}
+
+// stupid name
+type Bucketable interface {
+	Title() string // maybe this should be fmt.Stringer?
```

Comment:
The intention for this is to return debug data, so it could potentially just be a String()?